app/util/cmr.ts

```diff
@@ -254,6 +254,22 @@ export interface CmrPermissionsResponse extends CmrResponse {
   data: CmrPermissionsMap;
 }
 
+export interface CmrUmmCollection {
+  meta: {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    'concept-id': string;
+  };
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  umm: any;
```

Comment:
It's useful to list out the fields that we use for documentation reasons. So this wouldn't be the full list of UMM-C fields, just the ones harmony reads.

So it would include something like `ArchiveAndDistributionInformation { FileArchiveInformation:  { FormatType: string; Format: string;}`